### PR TITLE
Additions

### PR DESCRIPTION
**Pull request by Misfiy**
## Description
**Describe the changes** 
* Add ``Enums::HazardType``
* Add ``Extensions::BitwiseExtensions``
* Add ``Hazard::Type``
* Add ``Hazard:Get(HazardType)``
* Add ``PrefabHelper::GetPrefab<T>(PrefabType)``
* Add ``Server::MaxTps``
* Add ``TpsCommand``
* Add ``FpcRole::ClientRelativePosition`` and fix ``FpcRole::RelativePosition``

**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior?** (if this is a feature change)

**Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
No breaking changes made.

**Other information**:

<br />

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [X] Documentations
<br />

## Submission checklist

- [X] I have checked the project can be compiled
- [ ] I have tested my changes and it worked as expected

### Patches (if there are any changes related to Harmony patches)
- [ ] I have checked no IL patching errors in the console

### Other
- [X] Still requires more testing

### CODE REVIEW

File: EXILED/Exiled.API/Features/Hazards/AmnesticCloudHazard.cs [hunk 1]
```diff
@@ -26,9 +29,26 @@ public AmnesticCloudHazard(Scp939AmnesticCloudInstance hazard)
...
+
+                return amnesticCloudPrefab;
+            }
+        }
+
         /// <inheritdoc cref="Hazard.Base"/>
         public new Scp939AmnesticCloudInstance Base { get; }
 
+        /// <inheritdoc />
+        public override HazardType Type { get; } = HazardType.AmnesticCloud;
```
Comment (louis1706) [inline EXILED/Exiled.API/Features/Hazards/AmnesticCloudHazard.cs:50]: Not { get; } = but => for this

File: EXILED/Exiled.API/Features/Hazards/SinkholeHazard.cs [hunk 2]
```diff
@@ -28,5 +29,8 @@ public SinkholeHazard(SinkholeEnvironmentalHazard hazard)
         /// Gets the <see cref="SinkholeEnvironmentalHazard"/>.
         /// </summary>
         public new SinkholeEnvironmentalHazard Base { get; }
+
+        /// <inheritdoc />
+        public override HazardType Type { get; } = HazardType.Sinkhole;
```
Comment (louis1706) [inline EXILED/Exiled.API/Features/Hazards/SinkholeHazard.cs:34]: Same as above

File: EXILED/Exiled.API/Features/Hazards/TantrumHazard.cs [hunk 3]
```diff
@@ -26,11 +30,28 @@ public TantrumHazard(TantrumEnvironmentalHazard hazard)
...
+            }
+        }
+
         /// <summary>
         /// Gets the <see cref="TantrumEnvironmentalHazard"/>.
         /// </summary>
         public new TantrumEnvironmentalHazard Base { get; }
 
+        /// <inheritdoc />
+        public override HazardType Type { get; } = HazardType.Tantrum;
```
Comment (louis1706) [inline EXILED/Exiled.API/Features/Hazards/TantrumHazard.cs:53]: Same above as

**Review by louis1706 — CHANGES_REQUESTED** [review]

**Review by louis1706 — CHANGES_REQUESTED** [review]
Waiting for #17 to be merged for making more suggestions 

Use bitwise method when possible like i already said you on your previous PR

**Review by Misaka-ZeroTwo — COMMENTED** [review]
Fix conflicts

**Review by louis1706 — APPROVED** [review]

**Review by VALERA771 — APPROVED** [review]

**Pull request merged**